packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/translations/Messages_en_GB.json

```diff
@@ -15,7 +17,6 @@
   "kube_nodes_flavor_error": "An error has occurred loading the flavor",
   "kube_nodes_project_error": "An error has occurred loading the Public Cloud project",
   "kube_nodes_description_project_1": "Your Kubernetes cluster uses nodes to host your containers and services. All of your nodes will be launched in the {{ description }} Public Cloud project (you will be able to change this in the future).",
-  "kube_nodes_description_project_2": "These instances are deployed with standard hourly billing. You can find details on your bill and switch to a monthly plan in your Public Cloud project.",
```

Comment:
Removed in https://github.com/ovh/manager/commit/32427658e50456a45c5ff7badc95dadfdd7c17e8#diff-84bc22f1d1ac80a3964ebbd32875db16L18